client/www/pages/docs/patterns.md

```diff
@@ -387,6 +387,55 @@ function Login() {
 }
 ```
 
+## Managing local vs production apps
+
+You may want to have separate Instant apps for local development and production.
+The way to do this right now is to have two separate apps, one for local and one
+for production. You can then use an environment variables to switch between the two
+apps.
+
+```javascript
+// lib/db.ts
+import { init } from '@instantdb/react';
+
+// Next.js example
+const APP_ID =
+  process.env.NODE_ENV === 'production'
```

Comment:
I think the more common thing for people to do is to just override `NEXT_PUBLIC_INSTANT_APP_ID` in .env.local

That way you don't have to have separate env var names for the app id.